helper/labelSchema.js

```diff
@@ -13,12 +13,17 @@ module.exports = {
   'USA': {
     'borough': getFirstProperty(['borough']),
     'local': getFirstProperty(['locality', 'localadmin', 'county']),
-    'regional': getUsOrCaState,
+    'regional': getUsOrCaOrAusState,
     'country': getUSACountryValue
   },
+  'AUS': {
+    'local' : getFirstProperty(['locality', 'localadmin']),
+    'regional' : getUsOrCaOrAusState,
```

Comment:
There is a reason! It's a bit confusing. The labelGenerator.js module which actually generates the labels loads this module, and [picks the correct country's schema to use](https://github.com/pelias/api/blob/master/helper/labelGenerator.js#L41-L48), and then calls [the function from that schema](https://github.com/pelias/api/blob/master/helper/labelGenerator.js#L12-L13)